### PR TITLE
setup tekton chains instance deployment on op1st

### PR DESCRIPTION
**Pull request by harshad16**
feat: setup tekton chains instance deployment on op1st
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: 
- https://github.com/operate-first/support/issues/438
- https://github.com/operate-first/apps/pull/1278

## Description

- [x] Tekton chains provide security to our pipeline supply chains.
It would sign the images based on the secrets.
https://github.com/tektoncd/chains#installation

- [x] Updated the list of OWNERS.

### CODE REVIEW

File: tekton-chains/overlays/moc/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```
Comment (HumairAK) [inline tekton-chains/overlays/moc/kustomization.yaml:3]: hrmm I see there's a `base`...but it's not being used, is that intentional?

Comment (harshad16) [reply tekton-chains/overlays/moc/kustomization.yaml:3]: oh i missed it, thanks for the catch.

**Comment by harshad16** [comment]
Ready for the Re-Review

**Review by HumairAK — APPROVED** [review]
/lgtm
/approved

**Pull request merged**